### PR TITLE
WIP: feature/harta

### PR DESCRIPTION
**Pull request by codexms**
this PR contains `echarts` integration with Romain's counties map, and a new page `/harta` that should display the map and the candidates for a selected county.

There are some details that we need to decide before merging in the main branch. I wrote a couple of them bellow.

[1] Where is the best place to display the map, in a separate page or in main page?
[2] What details do we need tu display when the user interacts with the map (total nr of candidates f rom that county, what else)
[3] What information we need to fill in the cms beside the county code in order to support this feature
... anything else

### CODE REVIEW

File: lib/uitls.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import counties from "./counties.json"
```
Comment (cezarneaga) [inline lib/uitls.ts]: fisierierul asta suna a uituc. rename => utils.ts

File: pages/harta/[slug].tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                url: `https://nutotei.ro/harta`,
+                title: `Harta - Nu tot ei!`,
+                description: `Harta pe judete`,
+              }}
+            />
+            <div className="relative pt-6">
+              <main className="mt-16 mx-auto max-w-7xl px-4 sm:mt-24">
+                <div className="text-center">
+                  <h1 className="text-4xl tracking-tight font-extrabold text-gray-900 sm:text-5xl md:text-6xl">
+                    <span className="inline">Canditați </span>
```
Comment (cezarneaga) [inline pages/harta/[slug].tsx]: Hai sa incepem cu politruci. `Politruci `

File: components/latest-county.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import { CandidateCardBig } from "components/candidate-card-big";
+
+export function LatestCounty({ candidates, countyName }: { candidates: Candidate[], countyName: string }) {
+  return (
+    <div className="bg-white">
+      <div className="mx-auto py-12 px-4 max-w-7xl sm:px-6 lg:px-8 lg:py-16">
+        <div className="space-y-12 lg:grid lg:grid-cols-3 lg:gap-8 lg:space-y-0">
+          <div className="space-y-5 sm:space-y-4">
+            <h2 className="text-3xl font-extrabold tracking-tight sm:text-4xl">
+              Candidați din  <span className="text-red-600 inline">{countyName}</span>
```
Comment (cezarneaga) [inline components/latest-county.tsx]: Politruci din...

File: components/latest-county.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            </h2>
+            <p className="text-xl text-gray-500">
+              Postăm în fiecare săptămână profiluri de politicieni. Timpul ne
```
Comment (cezarneaga) [inline components/latest-county.tsx]: E important să știm pe cine votăm la noi acasă. Chiar fiind bine intenționați, putem promova oameni pe care nu îi vrem.

File: components/latest-county.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+              omoară, că material de descoperit avem!
+            </p>
+          </div>
+          <div className="lg:col-span-2">
+            <ul className="space-y-12 sm:grid sm:grid-cols-2 sm:gap-x-6 sm:gap-y-12 sm:space-y-0 lg:gap-x-8">
+              {candidates.map((candidate) => (
```
Comment (cezarneaga) [inline components/latest-county.tsx:20]: aici as face un split ca si pe partid intre carduri mari si mici, si paginare. dar pt acum e ok.

File: components/nav.tsx [hunk 6]
```diff
@@ -149,15 +147,24 @@ export function Nav() {
...
-                } hover:text-gray-900 dark:hover:text-gray-100`}
+                className={`text-base cursor-pointer font-medium ${activeRoute === "/despre"
+                  ? "text-gray-900 dark:text-white"
+                  : "text-gray-500 dark:text-gray-200"
+                  } hover:text-gray-900 dark:hover:text-gray-100`}
               >
                 Despre
               </span>
             </Link>
+            <Link href="/harta/bucuresti">
```
Comment (cezarneaga) [inline components/nav.tsx]: muta-l inainte la despre si vezi ca trebuie adaugat si la mobile nav.

File: components/counties-map/index.tsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+
+  const onEvents = {
+    'click': onClick,
+  }
+
+  return (data &&
+    <ReactEcharts
+      option={getChartOptions(data)}
+      style={{ height: '600px' }}
```
Comment (cezarneaga) [inline components/counties-map/index.tsx]: pe mobil e cam mare 600. nu merge sa wrap ReactEcharts intr-un div sai dai la ala height responsive cu tailwind si style={{height: '100%'}}

**Review by cezarneaga — CHANGES_REQUESTED** [review]
am facut niste comentarii. daca esti ok si le rezolvi eu zic ca putem face merge

File: components/latest-county.tsx (hunk 5 above)
Comment (codexms) [reply components/latest-county.tsx:20]: noted, pun un issue pt asta sa nu uitmam

File: lib/uitls.ts (hunk 1 above)
Comment (codexms) [reply lib/uitls.ts]: done

Comment (codexms) [reply lib/uitls.ts]: done

File: components/latest-county.tsx (hunk 3 above)
Comment (codexms) [reply components/latest-county.tsx]: done

File: components/counties-map/index.tsx (hunk 7 above)
Comment (codexms) [reply components/counties-map/index.tsx]: done

File: components/nav.tsx (hunk 6 above)
Comment (codexms) [reply components/nav.tsx]: done

File: components/latest-county.tsx (hunk 4 above)
Comment (codexms) [reply components/latest-county.tsx]: done

File: pages/harta/[slug].tsx (hunk 2 above)
Comment (codexms) [reply pages/harta/[slug].tsx]: done

**Pull request merged**